### PR TITLE
Milestone 4: Add desktop version

### PR DESCRIPTION
**Pull request by isume295**
In this milestone I: 

- Added responsive style for the desktop version of the project using media query.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -372,6 +383,7 @@ input:focus {
 
 @media (min-width: 375px) {
   .heroMainContainer {
+    /* height: 70%; */
```
Comment (huseyinbiyik) [inline styles.css]: Please do not leave any uncommented code in your codebase. In this way, you can keep your codebase clean, neat, organized and professional. If you want to access a code you used before, you can navigate through your git history with ```git checkout <commitnumber>```

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @isume295 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Design implemented very well 💯 
- Media queries used well 👍 
- Utilized from flex and grid to create responsiveness 👍 
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
⚠️  Please move your images/icons into a folder. It is a beautiful project but files create a mess. In this way, you can have a perfectly organized professional repository 👍   

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (isume295) [reply styles.css]: Thank you, I will remove the commented code

**Review by Zuhaib042 — APPROVED** [review]
Hi @isume295 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
✅ Good Implementation of suggested changes

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**